Forms/main.css

```diff
@@ -60,6 +87,7 @@ aside {
     width: 200px;
     float: right;
```

Comment:
I still didn't get the point of using `float: right`. Could you please provide screenshots of it working?